<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkIGTLDeviceSetupConnectionWidgetControls</class>
 <widget class="QWidget" name="QmitkIGTLDeviceSetupConnectionWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="4" column="0">
    <widget class="QPushButton" name="butConnect">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Connect with the host/Start server</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Server-IP</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editPort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Enter the port number of the host</string>
       </property>
       <property name="text">
        <string>18944</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editIP">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Enter the IP address of the host</string>
       </property>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="logIncomingMsg">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Enable this checkbox to log the send and receive message events</string>
       </property>
       <property name="text">
        <string>Log Incoming Messages</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="bufferOutMsgCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Buffer Outgoing Messages</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="bufferInMsgCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>If this checkbox is set the device stores all incoming messages in the queue. If it is not set it always overwrites the current value.</string>
       </property>
       <property name="text">
        <string>Buffer Incoming Messages</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="logOutgoingMsg">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Log Outgoing Messages</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
